src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourcelinkTests.cs

```diff
@@ -26,24 +35,40 @@ public class SourcelinkTests : SdkTests
     [SkippableFact(Config.SourceBuiltArtifactsPathEnv, skipOnNullOrWhiteSpaceEnv: true)]
     public void VerifySourcelinks()
     {
-        if (Directory.Exists(SourcelinkRoot))
+        try
         {
-            Directory.Delete(SourcelinkRoot, true);
-        }
-        Directory.CreateDirectory(SourcelinkRoot);
+            if (Directory.Exists(SourcelinkRoot))
+            {
+                Directory.Delete(SourcelinkRoot, true);
+            }
+            Directory.CreateDirectory(SourcelinkRoot);
 
-        IList<string> failedFiles = ValidateSymbols(ExtractSymbolsPackages(GetAllSymbolsPackages()), InitializeSourcelinkTool());
+            string symbolsRoot = Directory.CreateDirectory(Path.Combine(SourcelinkRoot, "symbols")).FullName;
 
-        if (failedFiles.Count > 0)
-        {
-            OutputHelper.WriteLine($"Sourcelink verification failed for the following files:");
-            foreach (string file in failedFiles)
+            // We are validating dotnet-symbols-*.tar.gz which contains all source-built symbols, including
```

Comment:
This comment needs updating with the recent rename.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17477/commits/fd4678ef74fb8870f6344668c765b4da5543fc35